hibike/virtual_device.py

```diff
@@ -89,19 +105,22 @@ def main():
         if msg.get_message_id() in [hm.MESSAGE_TYPES["SubscriptionRequest"]]:
             # Update the delay, subscription time,
             # and params, then send a subscription response
-            print("Subscription request recieved")
+            if args.verbose:
```

Comment:
Hmm i feel like we could restructure this to get rid of arg.verbose check multiple times.
Eh probably a future pull request